Engine/debug/debug.cpp

```diff
@@ -108,13 +108,41 @@ void send_message_to_debugger(const std::vector<std::pair<String, String>>& tag_
     editor_debugger->SendMessageToEditor(messageToSend.GetCStr());
 }
 
+String ToString(int v)
```

Comment:
There's a common function in StrUtils that does this:
https://github.com/adventuregamestudio/ags/blob/24fdb20eadf56f9a6d9a432d15491f8741540623/Common/util/string_utils.h#L40